tools/kessel-debug-container/scripts/zed-wrapper.sh

```
@@ -4,7 +4,7 @@
 # This provides a security policy layer for debug/read-only containers
 
 # Check if the any arguments provided are for write operations
-if [[ "$@" =~ "write" ]] || [[ "$@" =~ "delete" ]]; then
+if [[ "$@" =~ "write" ]]  || [[ "$@" =~ "create" ]] || [[ "$@" =~ "delete" ]]; then
```

Shellcheck (0.11.0)

[error] Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).

(SC2199)